frontend/src/metabase/query_builder/components/QueryModals.tsx

```diff
@@ -40,6 +45,10 @@ const mapDispatchToProps = {
   setQuestionCollection: Questions.actions.setCollection,
 };
 
+const mapStateToProps = (state: State) => ({
+  originalQuestionWithParameters: getOriginalQuestionWithParameters(state),
```

Comment:
We might not need this selector (or any of the new code associated with it) since models don't allow variables, just need some confirmation in case I'm missing something